### PR TITLE
Fix the bug that the max bandwidth is ignored if set before starting a meeting

### PR DESCRIPTION
**Pull request by simmkyu**
**Issue #:**
https://github.com/aws/amazon-chime-sdk-js/issues/1946

**Description of changes:**
- If the builder calls `chooseVideoInputQuality` API before starting a meeting, the SDK does not apply the max bandwidth to a local video.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
I created a minimal Chime SDK app to test the scenario.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ltrung** [comment]
Can we link to this issue https://github.com/aws/amazon-chime-sdk-js/issues/1946

File: test/audiovideocontroller/DefaultAudioVideoController.test.ts [hunk 1]
```diff
@@ -825,12 +826,15 @@ describe('DefaultAudioVideoController', () => {
       await delay(defaultDelay);
       expect(sessionStarted).to.be.true;
       expect(sessionConnecting).to.be.true;
-      audioVideoController.setVideoMaxBandwidthKbps(100);
       audioVideoController.handleHasBandwidthPriority(false);
       audioVideoController.handleHasBandwidthPriority(true);
       audioVideoController.handleHasBandwidthPriority(true);
```
Comment (xuesichao) [inline test/audiovideocontroller/DefaultAudioVideoController.test.ts:831]: This is not related to this PR. But I'm curious why we need two of this?

Comment (simmkyu) [reply test/audiovideocontroller/DefaultAudioVideoController.test.ts:831]: Looking at the `expect` lines. I don't think calling `handleHasBandwidthPriority` multiple times is not required. It might be for idempotence check.
